recipe/meta.yaml

```diff
@@ -26,41 +25,44 @@ outputs:
       run:
         - python
         - soupsieve >=1.2
+      run_constrained:
+        - html5lib
+        - lxml
+      test:
+        imports:
+          - bs4
+          - bs4.builder
+        requires:
+          - pip
+        commands:
+          - pip check
 
   - name: bs4
     build:
       noarch: generic
     requirements:
+      host:
+        - python
+        - pip
+        - setuptools
+        - wheel
       run:
+        - python
         - {{ pin_subpackage('beautifulsoup4', max_pin="x.x.x") }}
```

Comment:
As mentioned on slack, I'm not sure what's the point in the `bs4` output?

Reply:
It's a common name for `import bs4`

Reply:
Right, so conda users will expect to be able to `conda install bs4`?

Reply:
Yes